server/src/test/java/com/broadcom/lsp/cobol/usecases/TestParagraphSectionDefined.java

```diff
@@ -0,0 +1,27 @@
+package com.broadcom.lsp.cobol.usecases;
+
+import com.broadcom.lsp.cobol.usecases.engine.UseCaseEngine;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Map;
+
+class TestParagraphSectionDefined {
```

Comment:
Please, provide a description of what do you check here and what is the expected behavior

Reply:
Done